pdns/Makefile.am

```diff
@@ -1366,5 +1369,20 @@ ixfrdist.service: ixfrdist.service.in
 systemdsystemunit_DATA += \
 	ixfrdist.service
 endif # TOOLS
-
 endif # HAVE_SYSTEMD
+
+analyze_srcs = $(filter %.cc, $(pdns_server_SOURCES))
+analyze_plists = $(analyze_srcs:%.cc=%.plist)
+
+if HAVE_STATIC_ANALYSIS
+AM_LDFLAGS += \
+	-export-symbols
```

Comment:
I don't get the point of adding `-export-symbols` to `LD_FLAGS` here? It makes the linking fail on my host because libtool expects a path to a file to follow this parameter.

Reply:
On my system building the plists fails without this flag

Reply:
Do you by chance mean `-export-dynamic`?